### PR TITLE
Fix minor typos and add some suggestions [ci skip]

### PR DESCRIPTION
**Pull request by fatosmorina**
This _Pull Request_ includes some minor typo fixes and suggestions at README files and some comments inside actual code.

### CODE REVIEW

File: actionview/RUNNING_UNIT_TESTS.rdoc [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 The easiest way to run the unit tests is through Rake. The default task runs
 the entire test suite for all classes. For more information, checkout the
-full array of rake tasks with "rake -T"
+full array of rake tasks with +rake -T+.
```
Comment (tjschuck) [inline actionview/RUNNING_UNIT_TESTS.rdoc]: This needs to use `<tt>`, not `+`, since it's more than one word: `<tt>rake -T</tt>`

**Comment by fatosmorina** [comment]
@tjschuck Thank you very much for your review. I pushed the change.

@sgrif It's been almost two months since this PR has been opened. Can you please take a look and let me know if there is something that I need to change? Thanks in advance.

**Comment by rafaelfranca** [comment]
Can you squash all your commits?

**Comment by fatosmorina** [comment]
@rafaelfranca I have created a new PR with these commits squashed: 
https://github.com/rails/rails/pull/31264

I will close this old one.

**Pull request closed without merging**